python_modules/dagster/dagster/_cli/api.py

```diff
@@ -699,26 +696,31 @@ def _execute_step_command_body(
     help="[INTERNAL] Retrieves current utilization metrics from GRPC server.",
     envvar="DAGSTER_ENABLE_SERVER_METRICS",
 )
+@python_pointer_options
 def grpc_command(
     port: Optional[int],
     socket: Optional[str],
     host: str,
     max_workers: Optional[int],
-    heartbeat: bool = False,
-    heartbeat_timeout: int = 30,
-    lazy_load_user_code: bool = False,
-    fixed_server_id: Optional[str] = None,
-    log_level: str = "INFO",
-    log_format: str = "colored",
-    use_python_environment_entry_point: Optional[bool] = False,
-    container_image: Optional[str] = None,
-    container_context: Optional[str] = None,
-    location_name: Optional[str] = None,
-    instance_ref=None,
-    inject_env_vars_from_instance: bool = False,
```

Comment:
These defaults are unnecessary and misleading-- `click` will always call the function with values for all args, so no default is ever used.